examples/model_assisted_labeling/video_mal.ipynb

```diff
@@ -1,346 +1,557 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    batch_datarows = [dr.uid for dr in list(dataset.export_data_rows())]
</pre><p>The list() is redundant since you are using list comprehension.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes, Thank you !</p>